content/guides/applications/web-app-integrations/configure.md

```diff
@@ -38,42 +38,32 @@ To configure the integration, follow the guidance below for each value.
-| Client Callback URL      | Handles additional callback requests from Box after the primary request with Popup Integrations. If the application specifies a file parameter in the REST method, the preliminary callback URL cannot originate from the client. As a result, a second request must be made from the client to your server so the server can send the necessary interface to the user. |
+| Field | Description |
+|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
+| Client Callback URL | Handles additional callback requests from Box after the primary request with Popup Integrations. If the application specifies a file parameter in the REST method, the preliminary callback URL cannot originate from the client. As a result, a second request must be made from the client to your server so the server can send the necessary interface to the user. |
+| Preliminary Callback URL | The URL where callback parameters are sent when the user accepts the prompt. In most cases it should be a URL that performs an API call on the application server, but it can be any endpoint configured to accept HTTP requests. |
+| User Experience | Determines whether the integration is a [Popup][pu] or [Server-Side][ssi] Integration. |
+| New window settings | Determines if the application opens in a new tab. |
 
 <!-- markdownlint-enable line-length -->
 
-### Integration Status
-
-- **Development**: The integration is visible and available only to application
- collaborators listed under the **General Settings** tab. This option is best
- used when the application is still in development and undergoing testing.
-- **Online**: The integration is visible and available to all Box users. This
-  option is best used when development is complete and the application is
-  ready to publish in the App Center.
-- **Maintenance**: The integration is visible and available only to application
- collaborators listed under the **General Settings** tab. This option is best
- used after the integration is published in the App Center, but needs to
- perform maintenance updates or troubleshoot issues. Use this option to
- temporarily take the integration offline for everyone except the application's
- collaborators.
-
 ### Callback Parameters
 
 The "Callback Parameters" section configures the parameters that Box sends to
```

Comment:
```suggestion
The **Callback Parameters** section configures the parameters that Box sends to
```

Reply:
I was also thinking that maybe we should add some more info in the `Callback Parameters` section on how the parameters are constructed including Method, Param Name, Param value. 
And maybe a short example? 🤔